plugins/README.md

```diff
@@ -0,0 +1,23 @@
+Adding a new plugin
+-------------------
+
+An extensible architecture allows for providing new plugin types (for reading
+and writing different firmware) as well as ways quirk their behavior.
+
+You can find more information about the architecture in the developers section
+of the [fwupd website](http://www.fwupd.org).
+
+If you have a firmware specification and would like to see support
+in this project, please file an issue and share the spec.  Patches are also
+welcome.
+
+Plugin interaction
+------------------
+Some plugins may be able to influence the behavior of other plugins.
+This includes things like one plugin turnin on a device, or providing missing
+metadata to another plugin.
+
+The ABI for these interactions is defined in:
+https://github.com/hughsie/fwupd/blob/master/plugins/fu-plugin-metadata-abi.h
+
```

Comment:
I think it makes sense to have the includes in `src/`, project-root or `.` -- otherwise the list of include paths gets a bit unwieldy. I think the other "API" is in /src so it logitcally belongs there. The README.md in `/plugins` 100% agree with.